sdk/core/Azure.Core/tests/ModelSerializationTests/DogListProperty.cs

```diff
@@ -191,5 +199,36 @@ public new void Serialize(Stream stream, SerializableOptions options = default)
             writer.Flush();
         }
         #endregion
+
+        internal class DogListPropertyConverter : JsonConverter<DogListProperty>
+        {
+            public DogListPropertyConverter()
+            {
+                //I would like to have a custom property in the JsonSerializerOptions class but this doesn't seem to be there
+                //we also can't have a parameterless ctor without making this converter class public
+                //IgnoreAdditionalProperties = ignoreAdditionalProperties;
+            }
+
+            //public bool IgnoreAdditionalProperties { get; }
+
+            public override DogListProperty Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+            {
+                return DeserializeDogListProperty(JsonDocument.ParseValue(ref reader).RootElement, ConvertOptions(options));
+            }
+
+            public override void Write(Utf8JsonWriter writer, DogListProperty value, JsonSerializerOptions options)
+            {
+                ((IUtf8JsonSerializable)value).Write(writer, ConvertOptions(options));
+            }
+
+            private SerializableOptions ConvertOptions(JsonSerializerOptions options)
+            {
+                return new SerializableOptions()
+                {
+                    //IgnoreAdditionalProperties = IgnoreAdditionalProperties,
```

Comment:
remove?

Reply:
Potentially, I wanted to leave it as a discussion point for next week on where we could set this setting.  JsonSerializerOptions doesn't have anything for this their recommended way is to have constructor params on your converter but we want our converters to be internal so that wouldn't work.  There also doesn't appear to be a raw dictionary of key/value pairs that would could use to set this.  It is a gap if we go with STJ.